### PR TITLE
[fix #257] fix changefeed metric show

### LINKED ISSUE #257: cdc: changefeed metric do not show
### 1. Describe the bug

1. changefeed metric do not show
image

### 2. Minimal reproduce step (Required)
1. startup upstream tikv and tikv-cdc
2. startup downstream tikv
3. create a changefeed
4. watch the metric

### 3. What did you see instead (Required)
1. show nothing in changefeed
### 4. What did you expect to see? (Required)
1. show keyspan count in changefeed

- TiKV CDC:

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?
 fix changefeed metric show

Issue Number: close #257
see issue#257

Problem Description: **TBD**

### What is changed and how does it work?
image

### Code changes

- Has exported function/method change
- Has exported variable/fields change
- Has interface methods change
- Has persistent data change
- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

### CODE REVIEW

**Comment by zeminzhou** [comment]
PTAL~

**Review by haojinming — APPROVED** [review]
LGTM

File: cdc/metrics/grafana/tikv-cdc.json [hunk 1]
```diff
@@ -1225,7 +1225,7 @@
           ],
           "timeFrom": null,
           "timeShift": null,
-          "title": "Table count maintained by owner",
+          "title": "KeySpan count maintained by owner",
           "transform": "timeseries_aggregations",
```
Comment (pingyu) [inline cdc/metrics/grafana/tikv-cdc.json:1229]:
```suggestion
          "title": "Keyspan count maintained by owner",
```

Suggest to use uniform case of name.
Rest LGTM.

Comment (zeminzhou) [reply cdc/metrics/grafana/tikv-cdc.json]: fixed, thanks~

**Pull request merged**